include/abb_librws/v1_0/rw/retcode.h

```diff
@@ -0,0 +1,28 @@
+#include <abb_librws/rws.h>
+#include <abb_librws/v1_0/rws_client.h>
+
+#include <string>
+
+
+namespace abb :: rws :: v1_0 :: rw
+{
+    using namespace rws::rw;
+}
+
+
+namespace abb :: rws :: v1_0 :: rw :: retcode
+{
+    /**
+     * \brief A function for checking description of diagnostic code.
+     *
+     * https://developercenter.robotstudio.com/api/rwsApi/rwretcode_get_return_codes.html
+     *
+     * \param client RWS client
+     * \param code code to be checked.
+     * 
+     * \return \a RWSResult containing the result.
+     *
+     * \throw \a RWSError if something goes wrong.
+     */
+    RWSResult getRetcodeInfo(RWSClient& client, int const& code);
```

Comment:
`RWSResult` is used at a lower level. when you interact with `RWSClient` directly and don't interpret the result. Here we want to interpret the result as a structure describing a return code.

```suggestion
    RetcodeInto getRetcodeInfo(RWSClient& client, int const& code);
```